.changeset/fluffy-jars-protect.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core-components': minor
+'@backstage/plugin-catalog-react': patch
+---
+
+Uses new Autocomplete component from core-components that aligns with Select component UI for consistent a dropdown UI for all catalog filters
```

Comment:
Perhaps we could split this changeset in two to have a better description for the changes in core-components